### DISCUSSION TITLE
Translation of "the other"

### DISCUSSION THREAD

**Discussion by ChrisPerrette** [tekinged/missing#412] [Q&A]
What is the best way to say "other," as in, "I will eat one, and you can eat the other [one]"?

I know that _lmuut_ can mean "another," as in, "Beskak a lmuut."
The word _ngodech_ can mean "other" in the sense of "not this," as in, "Ng diak a ngodech el rael."

Thank you.

**Comment by johnbent** [comment]
I'll be curious to hear other people's answers but I would personally say something like 'sel/tial chimong'. Or 'bitang' if the thing being discussed was the other half of something. If you split a betelnut in half for example or were distributing a pair of socks or something.

**Comment by smith-371** [comment]
"I will eat one, and you can eat the other [one]"

I'd say it like (but I am not Palauan))): 

Ak mo kolii a tang, me ke mo kolii a kuk tang.
